tests/partition/metrics_test.cc

```diff
@@ -73,7 +73,7 @@ class APartitionedHypergraph : public Test {
     context.partition.k = 2;
     context.partition.objective = Objective::cut;
     context.partition.mode = Mode::direct_kway;
-    context.local_search.algorithm = RefinementAlgorithm::twoway_fm;
+    context.local_search.algorithm = RefinementAlgorithm::twoway_fm_hyperflow_cutter;
```

Comment:
@larsgottesbueren  Could this change be causing the errors we see in https://github.com/kahypar/kahypar/issues/200 ?

Reply:
Ah never mind, this is only a unit test.

Reply:
This change is only intended to run the new delta functions in the metrics object through a test case :)